paddle/pten/core/dense_tensor.cc

```diff
@@ -439,8 +387,7 @@ template <typename T>
 inline T* DenseTensor::mutable_data(const paddle::platform::Place& place,
                                     size_t requested_size) {
   static_assert(std::is_pod<T>::value, "T must be POD");
-  return reinterpret_cast<T*>(mutable_data(
-      place, paddle::framework::DataTypeTrait<T>::DataType(), requested_size));
+  return reinterpret_cast<T*>(mutable_data(place, requested_size));
```

Comment:
infinite recursive call?

Reply:
I thought it was a non-template function called. I will confirm, thanks